packages/environment-check/src/types.ts

```diff
@@ -99,6 +99,7 @@ export interface EndpointResults {
     isSapUi5Repo?: boolean; // SAPUI5 repository service for deployment available
     isTransportRequests?: boolean; // Ability to retrieve available Transport Requests
     HTML5DynamicDestination?: boolean;
+    HTML5TimeoutDestination?: boolean;
```

Comment:
@longieirl why is it named `HTML5TimeoutDestination` and not `HTML5Timeout`

Reply:
Updated now, thanks.